AssertionError when download bundles from ngc

```
[2024-10-18T08:13:09.197Z] ======================================================================
[2024-10-18T08:13:09.197Z] FAIL: test_download_ngc (tests.test_bundle_download.TestDownload)
[2024-10-18T08:13:09.197Z] Test checking MONAI version from a metadata file.
[2024-10-18T08:13:09.197Z] ----------------------------------------------------------------------
[2024-10-18T08:13:09.197Z] Traceback (most recent call last):
[2024-10-18T08:13:09.197Z]   File "/usr/lib/python3.10/unittest/mock.py", line 1379, in patched
[2024-10-18T08:13:09.197Z]     return func(*newargs, **newkeywargs)
[2024-10-18T08:13:09.197Z]   File "/tmp/MONAI/tests/test_bundle_download.py", line 259, in test_download_ngc
[2024-10-18T08:13:09.197Z]     mock_logger.warning.assert_not_called()
[2024-10-18T08:13:09.197Z]   File "/usr/lib/python3.10/unittest/mock.py", line 890, in assert_not_called
[2024-10-18T08:13:09.197Z]     raise AssertionError(msg)
[2024-10-18T08:13:09.197Z] AssertionError: Expected 'warning' to not have been called. Called 1 times.
[2024-10-18T08:13:09.197Z] Calls: [call('Your MONAI version is 1.2, but the bundle is built on MONAI version 1.4.0.')].
```

--- a/monai/networks/utils.py
+++ b/monai/networks/utils.py
@@ -697,14 +697,14 @@ def convert_to_onnx(
         torch.onnx.export(
             mode_to_export,
             onnx_inputs,
-            f=f,
+            f=f,  # type: ignore[arg-type]
             input_names=input_names,
             output_names=output_names,
             dynamic_axes=dynamic_axes,
             opset_version=opset_version,
             do_constant_folding=do_constant_folding,
             **torch_versioned_kwargs,
-        )  # type: ignore[arg-type]
+        )
         if filename is None:
             onnx_model = onnx.load_model_from_string(f.getvalue())
         else:

--- a/monai/networks/utils.py
+++ b/monai/networks/utils.py
@@ -694,7 +694,7 @@ def convert_to_onnx(
         else:
             f = filename
 
-        torch.onnx.export(  # type: ignore[arg-type]
+        torch.onnx.export(
             mode_to_export,
             onnx_inputs,
             f=f,
@@ -704,7 +704,7 @@ def convert_to_onnx(
             opset_version=opset_version,
             do_constant_folding=do_constant_folding,
             **torch_versioned_kwargs,
-        )
+        )  # type: ignore[arg-type]
         if filename is None:
             onnx_model = onnx.load_model_from_string(f.getvalue())
         else:

--- a/monai/networks/utils.py
+++ b/monai/networks/utils.py
@@ -694,7 +694,7 @@ def convert_to_onnx(
         else:
             f = filename
 
-        torch.onnx.export(
+        torch.onnx.export(  # type: ignore[arg-type]
             mode_to_export,
             onnx_inputs,
             f=f,

--- a/tests/test_bundle_download.py
+++ b/tests/test_bundle_download.py
@@ -250,7 +250,7 @@ class TestDownload(unittest.TestCase):
                 mock_logger.warning.assert_called_once()
 
     @skip_if_quick
-    @patch("monai.bundle.scripts.get_versions", return_value={"version": "1.2"})
+    @patch("monai.bundle.scripts.get_versions", return_value={"version": "1.3"})
     def test_download_ngc(self, mock_get_versions):
         """Test checking MONAI version from a metadata file."""
         with patch("monai.bundle.scripts.logger") as mock_logger:

